### PR TITLE
CI test fixes

### PR DESCRIPTION
**Pull request by dennisreimann**
Trying to fix that damn test.

### CODE REVIEW

**Comment by bolatovumar** [comment]
lol it failed anyway

**Comment by woutersamaey** [comment]
Omg 😱

**Comment by dennisreimann** [comment]
@NicolasDorier This one downgrades NBXplorer again, see my [comments in the chat](https://chat.btcpayserver.org/btcpayserver/pl/3bdjfy76mfrmbqn531w8rau66w): There seems to be a regression exposed by the `CanPayWithTwoCurrencies` test. We can use this PR to also fix that one.

File: BTCPayServer.Tests/docker-compose.altcoins.yml [hunk 1]
```diff
@@ -89,11 +89,11 @@ services:
     expose:
       - "4444"
   nbxplorer:
-    image: nicolasdorier/nbxplorer:2.3.6
+    image: nicolasdorier/nbxplorer:2.2.18
```
Comment (dennisreimann) [inline BTCPayServer.Tests/docker-compose.altcoins.yml:92]: We should find a fix for the new NBXplorer version, downgrading won't help.

File: BTCPayServer.Tests/ApiKeysTests.cs [hunk 2]
```diff
@@ -19,7 +20,7 @@ namespace BTCPayServer.Tests
 {
     public class ApiKeysTests : UnitTestBase
     {
-        public const int TestTimeout = TestUtils.TestTimeout;
+        public const int TestTimeout = 120_000;
```
Comment (dennisreimann) [inline BTCPayServer.Tests/ApiKeysTests.cs:23]: As simple as it is, this is the actual fix. 

The default timeout of 60 seconds is too short. The test simply takes very long and given 60 seconds it erratically times out in various places. The test does A LOT of requests back and forth to test various API keys. Sometimes the latency seems to be heavy and in sum it takes rather long.

Comment (woutersamaey) [reply BTCPayServer.Tests/ApiKeysTests.cs:23]: Are we sure there isn't a problem with the API keys page which makes it take unusually long? I too have had the impression that the API page is slow, but it's just a feeling.

Comment (dennisreimann) [reply BTCPayServer.Tests/ApiKeysTests.cs:23]: @woutersamaey I've [tested this at length](https://app.circleci.com/pipelines/github/btcpayserver/btcpayserver?branch=pull%2F3609&filter=all) and there's not a single or clear reason why it fails (sometimes it just works). I think this is just a matter of the latency between requests on CI — you can run the test locally and it perfoms regularly like any other of our Selenium tests.

Comment (woutersamaey) [reply BTCPayServer.Tests/ApiKeysTests.cs:23]: Thanks for your hard work @dennisreimann !

**Pull request merged**